lib/plugins/create/templates/aws-csharp/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+#build handlers
+dotnet restore
+dotnet publish -c release
+
+#install zip
+apt-get -qq update
+apt-get -qq -y install zip
```

Comment:
I think we can skip those two lines and just assume that zip is already installed.